index.src.html

```diff
@@ -390,27 +393,34 @@ <h3 id="serialize-reports" algorithm>Serialize Reports</h3>
   <h2 id="document-reporting">Document Centered Reporting</h2>
 
   This section defines the mechanism for configuring reporting endpoints for
-  reports generated by actions in a document. Such reports have a lifetime which
-  is tied to the document where they generated.
+  reports generated by actions in a document (or in a worker script). Such
+  reports have a lifetime which is tied to that of the document or worker where
+  they were generated.
 
   <h3 id="document-configuration">Document configuration</h3>
 
-  Each <a>document</a> has an <dfn for="document" export
-  attribute>endpoints</dfn> list, which is a list of <a>endpoints</a>, each of
-  which MUST have a distinct {{endpoint/name}}.  (The algorithm in
-  [[#process-header]] guarantees this by keeping the first entry in a
-  `Reporting-Endpoints` header with a particular name.)
+  Each object implementing {{WindowOrWorkerGlobalScope}} has an <dfn
+  for="WindowOrWorkerGlobalScope" export>endpoints</dfn> list, which
+  is a list of <a for="/">endpoints</a>, each of which MUST have a distinct
+  {{endpoint/name}}.  (Uniqueness is guaranteed by the algorithm in
+  [[#process-header]].)
 
-  A server MAY define a set of reporting endpoints for a resource it returns,
-  via the <a>`Reporting-Endpoints`</a> HTTP response header field. This
-  mechanism is defined in [[#header]], and its processing in
-  [[#process-header]].
+  Each object implementing {{WindowOrWorkerGlobalScope}} has an <dfn
+  for="WindowOrWorkerGlobalScope" export>reports</dfn> list, which is
+  a list of <a for="/">reports</a>.
 
-  Each <a>document</a> has an <dfn for="document" export attribute>reports</dfn>
-  list, which is a list of <a>reports</a>.
+  To <dfn export>initialize a document or worker's endpoint list</dfn>, given a
```

Comment:
Should this be "initialize a global's endpoint list" perhaps?

Reply:
I'm assuming the idea is to patch HTML to run this algorithm on global object creation?

Reply:
Yes, that's in progress in https://github.com/whatwg/html/pull/5470

Reply:
Ping on

> Should this be "initialize a global's endpoint list" perhaps?

Reply:
> Should this be "initialize a global's endpoint list" perhaps?

I missed that one; that's a better name 👍